src/spells.cpp

```diff
@@ -702,7 +702,7 @@ bool Spell::playerRuneSpellCheck(Player* player, const Position& toPos)
 		return false;
 	}
 
-	const Creature* topVisibleCreature = tile->getBottomVisibleCreature(player);
+	auto topVisibleCreature = tile->getBottomVisibleCreature(player);
```

Comment:
Could this be `const auto`
```suggestion
	const auto topVisibleCreature = tile->getBottomVisibleCreature(player);
```